.changeset/swift-gorillas-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/app-admin-ui': patch
+---
+
+Correctly load Content field views when field access set to update: false
```

Comment:
Is this the description for the other PR?

Reply:
Nope, that's correct. Without this fix, the Content field will not load if you set `update: false` - it'll throw an error about deserialisation being called before it was loaded.

The _other_ issue is https://github.com/keystonejs/keystone/issues/2150 which is how fields with `update: false` on them will just disapear completely from the Admin UI